src/patternfly/demos/Banner/__banner-table.hbs

```diff
@@ -0,0 +1,183 @@
+{{#> table table--id=(concat page--id '-table') table--grid="true" table--modifier="pf-m-grid-xl" table--attribute='aria-label="This is a table with checkboxes"'}}
```

Comment:
This is what the demo looks like right now,  i dont know if the banner is working

<img width="694" alt="Screen Shot 2020-06-17 at 9 34 44 AM" src="https://user-images.githubusercontent.com/20118816/84905038-1b620d00-b07e-11ea-932b-20ae1f4c0e88.png">